src/main/resources/db/migration/V7.0__DDL_modify_employee_role_table.sql

```diff
@@ -0,0 +1,34 @@
+alter table "position"
+    rename to "role";
+
+CREATE TABLE employee_role(
+    employee_id  UUID NOT NULL,
+    role_id UUID NOT NULL,
+    FOREIGN KEY (employee_id) REFERENCES employee(id),
+    FOREIGN KEY (role_id) REFERENCES "role"(id)
+);
+
+ALTER TABLE employee
+    RENAME COLUMN email TO personal_email;
+
+ALTER TABLE employee
+    add emergency_contact_full_name VARCHAR(255);
+
+ALTER TABLE employee
+    add emergency_contact_phone VARCHAR(255);
+
+ALTER TABLE employee
+    add tax_id VARCHAR(255);
```

Comment:
change to `add tax_id VARCHAR(255) set not null; ` and remove the set not null below